guess-the-country/country-frontend/src/App.vue

```diff
@@ -164,6 +171,7 @@ export default {
       streetviewimage: null,
       explainimage: null,
       waitingForExplanation: false,
+      sequence_mode: process.env.VUE_APP_IMAGE_SEQUENCE_MODE,
```

Comment:
Es wäre super, wenn das in der README.md dokumentiert wäre.

Beispiel: https://github.com/XAI-Demonstrator/xai-demonstrator/tree/master/visual-inspection#environment-variables